Doc/library/gzip.rst

```diff
@@ -101,9 +101,12 @@ The module defines the following items:
    is no compression. The default is ``9``.
 
    The optional *mtime* argument is the timestamp requested by gzip. The time
-   is in Unix format, i.e., seconds since 00:00:00 UTC, January 1, 1970.
-   If *mtime* is omitted or ``None``, the current time is used. Use *mtime* = 0
-   to generate a compressed stream that does not depend on creation time.
+   is in Unix format, i.e., seconds since 00:00:00 UTC, January 1, 1970. Set
+   *mtime* to ``0`` to generate a compressed stream that does not depend on
+   creation time. If *mtime* is omitted or ``None``, the current time is used;
+   however, if the current time is outside the range 00:00:00 UTC, January 1,
+   1970 through 06:28:15 UTC, February 7, 2106, then the value ``0`` is used
+   instead.
 
```

Comment:
The docs are slightly wrong now that `GzipFile.__init__` will raise a ValueError if the specified value is out of range.
I know you've documented it in the docstring for `__init__`, but I think it's worth clarifying here since we're already in the area.